public/js/monitor.js

```diff
@@ -410,6 +438,34 @@ function addBentoObserver(){
       });
     }
   }
+  const researchPromo = document.querySelector(".research-recruitment-wrapper");
+  const sendRecruitmentPing = (action, interactionStatus) => {
+    if (gaAvailable) {
+      ga("send", "event", "Research Recruitment Promo", action, "Recruitment Promo", {nonInteraction: interactionStatus});
```

Comment:
You'll need to send additional options on the click/Engage event (but not the "view" event) 
`{transport: "beacon"}`

Similar to this: https://github.com/mozilla/blurts-server/blob/master/public/js/fxa-analytics.js#L75